src/go/configgenerator/listener_generator.go

```diff
@@ -799,3 +788,38 @@ func makeRouterFilter(opts options.ConfigGeneratorOptions) *hcmpb.HttpFilter {
 	}
 	return routerFilter
 }
+
+func transcodingIgnoreQueryParameters(serviceInfo *sc.ServiceInfo) []string {
```

Comment:
move this logic into serviceinfo.go, and considering to consolidate TranscoderIgnoredJwtQueryParams, TranscoderIgnoredApiKeyQueryParams, transcodingIgnoreQueryParameters.